modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,94 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify a standalone action can be triggered manually to send an email"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template and given the notification check interval set to 1") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			Click(
+				key_itemName = "Notifications",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			SystemSettings.editTextSetting(
+				settingName = "Notification Queue Entry Delete Interval",
+				settingValue = 1);
+
+			SystemSettings.saveConfiguration();
```

Comment:
@bronsonsoda so turns out you don't need to set this, since by default it'll send the email notification instantly. You can remove these steps. Can you also update the BDD step and remove the property to skip portal instances?

Reply:
Sorry about that